yarn.lock

```diff
@@ -3,6 +3,7 @@
 
 
 "@aashutoshrathi/word-wrap@^1.2.3", "@aashutoshrathi/word-wrap@^1.2.6", "word-wrap@npm:@aashutoshrathi/word-wrap@^1.2.6":
+  name word-wrap
```

Comment:
Is this necessary?

Reply:
yes, without which lint script is not working